tools/encyclopedia/macros.xml

```diff
@@ -3,6 +3,7 @@
     <xml name="requirements">
         <requirements>
             <requirement type="package" version="@VERSION@">encyclopedia</requirement>
+            <requirement type="package" version="1">fonts-conda-ecosystem</requirement>
```

Comment:
we should add this to the conda package, isn't it?

Reply:
yep. just want to test if it helps. one of the tools needs massive amounts of memory for the test .. crashing my laptop.

Reply:
I also needed to add openjdk 11 .. by default 10 was installed which misses some packages, e.g. fontconfig. tested successfully locally for encyclopedia

not sure whats the best way to fix it, just update the bioconda recipes in the same way?

Reply:
yes, I think that should work

Reply:
https://github.com/bioconda/bioconda-recipes/pull/25508